### ISSUE TITLE
MRE SDK link is down

### ISSUE THREAD

**Issue by azihassan** [UstadMobile/ustadmobile-mre#2]
The linked SDK (http://www.ustadmobile.com/mre/MRE_SDK_3.0.00.20_Normal_Eng_0.zip) results in a 404. Googling the filename yields nothing. Is there a working link for this SDK ?

**Comment by sankalpdeveloper** [comment]
Yes it is removed from his server bit I have copy of it on my google drive
I give you link 

> [https://drive.google.com/file/d/1KIh3VrOIAYLVw5cOf2_uPQvJbOmT6MyC/view?usp=drivesdk](url)

**Comment by sankalpdeveloper** [comment]
If you want any other software that are missing you may found here
[https://drive.google.com/folderview?id=12ohq-7V0_02QQX9Kjf1m6YRWHO9-JpPy](url)

**Comment by varunasingh** [comment]
Hey guys, 

This was an old project that we did and unfortunately had to stop investing time in. Working with MRE was not straight forward and cumbersome. 
Looking ahead, We're impressed with the direction that KaiOS is going - This sounds like a very promising feature phone operating system. https://www.kaiostech.com/
Quite a few new feature phones are adopting this. 

I can see if there is any other MRE related resources/documentation that I have.

**Comment by sankalp19991** [comment]
I have all tool in my drive link so you can download

**Comment by azihassan** [comment]
Thank you for the quick replies everyone, I appreciate it. I noticed the SDK is an exe file, is there a Linux-compatible version ?

@varunasingh it does indeed look promising. Unfortunately I'm stuck with an old phone that still uses MRE, so I'm trying to get my hands on anything that can help me develop apps for it. In that regard, this repo is a goldmine :+1:

**Comment by sankalpdeveloper** [comment]
> If you want any other software that are missing you may found here
> [https://drive.google.com/folderview?id=12ohq-7V0_02QQX9Kjf1m6YRWHO9-JpPy](url)

You can download all tool here

**Comment by sankalpdeveloper** [comment]
> Thank you for the quick replies everyone, I appreciate it. I noticed the SDK is an exe file, is there a Linux-compatible version ?
> 
> @varunasingh it does indeed look promising. Unfortunately I'm stuck with an old phone that still uses MRE, so I'm trying to get my hands on anything that can help me develop apps for it. In that regard, this repo is a goldmine

If you make any app can you send me at 
sankalpchoudhary8@gmail.com
I would like to test it

**Comment by sankalpdeveloper** [comment]
> Thank you for the quick replies everyone, I appreciate it. I noticed the SDK is an exe file, is there a Linux-compatible version ?
> 
> @varunasingh it does indeed look promising. Unfortunately I'm stuck with an old phone that still uses MRE, so I'm trying to get my hands on anything that can help me develop apps for it. In that regard, this repo is a goldmine

Sorry but when I download it has only windows version so no linux compatible version

**Comment by KielYam1212008** [comment]
You or/and he download and install program SDK. also known as Devkit:
https://archive.org/details/mre-sdk
Coming soon appears the ultities, programs, apps and files... in/on the Wayback Machine!

**Comment by raspiduino** [comment]
Thank you. You guys are awesome for preserving this SDK!

**Comment by sankalpdeveloper** [comment]
> You or/and he download and install program SDK. also known as Devkit: https://archive.org/details/mre-sdk Coming soon appears the ultities, programs, apps and files... in/on the Wayback Machine!

I think This is Down

**Comment by sankalpdeveloper** [comment]
Is anybody have link to download  Visual Studio2008 / Visual C++ 2008 Express

**Comment by raspiduino** [comment]
Yes, it has a try period for 90 days (after that you need to cr@ck :))


> Is anybody have link to download Visual Studio2008 / Visual C++ 2008 Express

Here is the direct link from Microsoft:
https://download.microsoft.com/download/8/1/d/81d3f35e-fa03-485b-953b-ff952e402520/VS2008ProEdition90dayTrialENUX1435622.iso

P/S: Don't use crack, try cracking it yourself. This is much safer and much more fun :) (if you have time)

**Comment by raspiduino** [comment]
I also made a backup of MRE SDK at https://github.com/raspiduino/mre-sdk in case the Google Drive down.

**Comment by raspiduino** [comment]
@sankalpdeveloper do you have MRE SDK 2.0? And also 1.0 if possible? Thanks!
